monthly-exams/exam-1/src/App.css

```diff
@@ -0,0 +1,83 @@
+body {
+  width: 100vw;
+  height: 100vh;
+  margin: 0;
+}
+
+.container {
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: white;
+}
+
+.button-row {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 20px;
+}
+
+.button {
+  border: 1px solid #ccc;
+  border-radius: 0;
+  background-color: black;
+  color: #fff;
+  padding: 10px 20px;
+  font-size: 20px;
+  margin: 5px;
+  cursor: pointer;
+}
+
+.button.red {
```

Comment:
Things to improve in your CSS:
1. There is no need to combine `.button` + `.red` like this.
2. Avoid repeating the same background-color like `.button.red` and `.container.red` as both selectors basically give you the same background.

Check my answer to see how I do the CSS